tests/numpy/test_stack.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+# Copyright (C) 2020 NumS Development Team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import numpy as np
+
+
+# pylint: disable=import-outside-toplevel
+
+
+def test_atleast_1d(nps_app_inst):
+    import nums.numpy as nps
+    assert nps_app_inst is not None
+
+    x = 1.0
+    assert np.allclose(nps.atleast_1d(x).get(), np.atleast_1d(x))
+
+    x = np.arange(9).reshape(3, 3)
+    assert np.allclose(nps.atleast_1d(x).get(), np.atleast_1d(x))
```

Comment:
While passing in a numpy array as-is is fine with the existing code, as I mention elsewhere, this creates a precedent that requires us to support such semantics for all operations (which is currently not true). It may make sense to provide support for auto conversion as a separate PR, with its own explicit set of tests, and explicitly convert the numpy arrays in this test to BlockArrays.

Reply:
Sounds good.